### PR TITLE
Adds fluff items

### PR DESCRIPTION
**Pull request by stobarico**
Adds/Replaces: https://i.imgur.com/egB0KUW.png (Sprites and icons gently borrowed from Paradise Station)

### CODE REVIEW

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -68,25 +68,22 @@
...
 	materials = list(MAT_GOLD = 750)
 	sharpness = IS_SHARP
 	resistance_flags = FIRE_PROOF
-	unique_reskin = list("Oak" = "pen-fountain-o",
-						"Gold" = "pen-fountain-g",
-						"Rosewood" = "pen-fountain-r",
-						"Black and Silver" = "pen-fountain-b",
-						"Command Blue" = "pen-fountain-cb"
+	unique_reskin = list("Gold" = "goldpen",
+						"Black and Silver" = "fancypen",
```
Comment (Qustinnus) [inline code/modules/paperwork/pen.dm]: why are they all removed?

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -244,7 +299,7 @@
 	desc = "A formal black suit and red tie, intended for the station's finest."
 	icon_state = "really_black_suit"
 	item_state = "bl_suit"
-	item_color = "black_suit"
+	item_color = "really_black_suit"
```
Comment (Qustinnus) [inline code/modules/clothing/under/miscellaneous.dm]: 🤔

File: code/modules/clothing/under/miscellaneous.dm [hunk 3]
```diff
@@ -159,6 +159,61 @@
 	item_state = "dg_suit"
 	item_color = "centcom"
 
+/obj/item/clothing/under/solgov
```
Comment (Qustinnus) [inline code/modules/clothing/under/miscellaneous.dm]: should make these replaces load-outs of boarding teams

File: code/modules/paperwork/pen.dm (hunk 1 above)
Comment (stobarico) [inline code/modules/paperwork/pen.dm]: No sprites for them, so, instead of doing what the bartender shotgun does, that it leaves the missing sprite loadout there so someone chooses it and it goes invisible, i decided to remove the spriteless ones

File: code/modules/clothing/under/miscellaneous.dm (hunk 3 above)
Comment (stobarico) [inline code/modules/clothing/under/miscellaneous.dm]: Can't we do that ingame? If so, i could make another PR

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (stobarico) [inline code/modules/clothing/under/miscellaneous.dm]: [https://i.imgur.com/pTbA158.png](url)

File: code/modules/paperwork/pen.dm (hunk 1 above)
Comment (Qustinnus) [inline code/modules/paperwork/pen.dm]: why are the sprites missing

File: code/modules/clothing/under/miscellaneous.dm (hunk 3 above)
Comment (Qustinnus) [inline code/modules/clothing/under/miscellaneous.dm]: combine it with this PR

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (Qustinnus) [inline code/modules/clothing/under/miscellaneous.dm]: broken url

File: code/modules/paperwork/pen.dm (hunk 1 above)
Comment (stobarico) [inline code/modules/paperwork/pen.dm]: The cap's pen is a TG thing, and apparently we didn't have any fancy pen sprites, so, i ported the ones from paradise station, and those were the pen sprites that were possible.

Comment (Qustinnus) [inline code/modules/paperwork/pen.dm]: can you also get TGs sprites and add the code back?

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (Qustinnus) [inline code/modules/clothing/under/miscellaneous.dm]: well ok i guess?

File: code/modules/clothing/under/miscellaneous.dm (hunk 3 above)
Comment (stobarico) [inline code/modules/clothing/under/miscellaneous.dm]: SolGov outfits modified
